redaxo/src/addons/phpmailer/package.yml

```
@@ -19,4 +19,4 @@ pages:
         perm: admin
 
 requires:
-    redaxo: ^5.4.0
+    redaxo: ^5.7.0
```

```suggestion
    redaxo: ^5.7.0-beta1
```

Da 5.7.0 > 5.7.0-beta1, und somit es sonst nicht lauffähig wäre mit der Beta.

---

Danke